circuits/circom/test/circuit.test.ts

```diff
@@ -0,0 +1,108 @@
+import { Point, sign } from '@noble/secp256k1'
+import { wasm as wasm_tester } from 'circom_tester'
+import { buildPoseidon } from 'circomlibjs'
+import { BigNumber, utils } from 'ethers'
+import { Scalar } from 'ffjavascript'
+
+import { join } from 'path'
+import {
+  bigintToArray,
+  bigintToUint8Array,
+  uint8ArrayToBigint,
+} from './helpers'
+
+// FIXME: unused?
+exports.p = Scalar.fromString(
```

Comment:
why are we exporting something here?

Reply:
Honestly no idea. It's a copy/paste from [here](https://github.com/iden3/circomlib/blob/243d2ec4fc9855780632fc498d24415b5534019f/test/escalarmulany.js#L7).